src/common/options/global.yaml.in

```diff
@@ -2984,15 +2984,15 @@ options:
   type: int
   level: advanced
   desc: How often (in seconds) should PGs stats be collected.
-  with_legacy: false
+  with_legacy: true
   default: 5
 - name: osd_pg_stat_report_interval_max_epochs
   type: int
   level: advanced
   desc: The maximum number of epochs allowed to pass before PG stats
         are collected.
   default: 500
-  with_legacy: false
+  with_legacy: true
```

Comment:
Officially, we do not add new legacy options:

https://github.com/ceph/ceph/blob/a8e43a9aa1fa9ac78fec8049aef1ff18aeff789f/src/common/config_values.h#L36-L40

You can load these via `OSD::handle_conf_change`.

Reply:
Ouch, moved to `md_config_cacher_t`.